### DISCUSSION TITLE
UZG-01 connected to iobroker has low connectivity to zigbee devices

### DISCUSSION THREAD

**Discussion by averlon** [xyzroe/XZG#103] [Q&A]
Hi,
I have installed the UZG-01 in my Network (via RJ45) in my house.
The device is placed in a central location in the house (basement, ground floor, first floor). So I have placed the device in the ground floor somewhere close the staircase.

I then installed zigbee switches in the staircase area in the basement.

Connectivity is very very bad. I can switch the device in the zigbee adapter, it switches sometimes, sometines not, although shown in the zigbee adapter of iobroker as "switched".

How can I increase transmission to enable better connection to zigbee devices?

**Comment by xyzroe** [comment]
The first thing you need to do is to scan your radio airwaves and choose the right channel for your Zigbee network. A lot depends on it. 

If you have done this and the problem persists, you probably have a hardware problem, I would contact the seller.
